docker-compose.yaml

```diff
@@ -106,6 +107,9 @@ services:
       - DJANGO_SUPERUSER_PASSWORD=admin
       - LOG_LEVEL=DEBUG
       - NOTIFICATIONS_DISABLED=true
+      - OPENZAAK_NOTIF_CONFIG_ENABLE=true
```

Comment:
We should change the PR description to reflect the fact we're bumping keycloak *and* open-zaak versions